cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -1,16 +1,17 @@
 define([
-    'underscore', 'js/models/course', 'js/models/group_configuration', 'js/models/group',
+    'underscore', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers', 'common/js/spec_helpers/template_helpers',
+    'js/models/course', 'js/models/group_configuration', 'js/models/group',
     'js/collections/group_configuration', 'js/collections/group',
     'js/views/group_configuration_details', 'js/views/group_configurations_list', 'js/views/group_configuration_editor',
     'js/views/group_configuration_item', 'js/views/experiment_group_edit', 'js/views/content_group_list',
     'js/views/content_group_details', 'js/views/content_group_editor', 'js/views/content_group_item',
-    'common/js/components/views/feedback_notification', 'common/js/spec_helpers/ajax_helpers', 'common/js/spec_helpers/template_helpers',
+    'common/js/components/views/feedback_notification',
     'common/js/spec_helpers/view_helpers'
 ], function(
-    _, Course, GroupConfigurationModel, GroupModel, GroupConfigurationCollection, GroupCollection,
-    GroupConfigurationDetailsView, GroupConfigurationsListView, GroupConfigurationEditorView,
+    _, AjaxHelpers, TemplateHelpers, Course, GroupConfigurationModel, GroupModel, GroupConfigurationCollection, 
+    GroupCollection, GroupConfigurationDetailsView, GroupConfigurationsListView, GroupConfigurationEditorView,
     GroupConfigurationItemView, ExperimentGroupEditView, GroupList, ContentGroupDetailsView,
-    ContentGroupEditorView, ContentGroupItemView, Notification, AjaxHelpers, TemplateHelpers, ViewHelpers
+    ContentGroupEditorView, ContentGroupItemView, Notification, ViewHelpers
```

Comment:
If we move `TemplateHelpers` up front to be with `AjaxHelpers`, whe should probably move `ViewHelpers` as well for consistency

(Unless this was a change because of an actual load order issue and not just for style?)